source/operations/server-side-encryption/configure-minio-kes.rst

```diff
@@ -0,0 +1,220 @@
+.. _minio-sse-vault:
+.. _minio-sse-gcp:
+.. _minio-sse-azure:
+.. _minio-sse-aws:
+
+======================================
+Server-Side Object Encryption with KES
+======================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. |EK|            replace:: :abbr:`EK (External Key)`
+.. |SSE|           replace:: :abbr:`SSE (Server-Side Encryption)`
+.. |KMS|           replace:: :abbr:`KMS (Key Management System)`
+.. |KES-git|       replace:: :minio-git:`Key Encryption Service (KES) <kes>`
+.. |KES|           replace:: :abbr:`KES (Key Encryption Service)`
+.. |rootkms|       replace:: `Hashicorp Vault <https://vaultproject.io/>`__
+.. |rootkms-short| replace:: Vault
+
```

Comment:
Lets add

```suggestion

.. meta::
   :description: Deploy MinIO with Server-Side Object Encryption
   :keywords: encryption, security, hashicorp, keyvault, azure

```

Apparently Google ignores a lot of stuff so this should be interesting.

Reply:
done